src/hotspot/share/gc/shenandoah/shenandoahLogFileOutput.cpp

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright (c) 2021, Amazon.com, Inc. All rights reserved.
+ * Copyright (c) 2015, 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Is this a copy of or derived from an existing source file? If not, only putting the 2021 header should be enough.